### ISSUE TITLE
Why are Major/Minor Process Lines and other process related equipment not modelled?

### ISSUE THREAD

**Issue by CocoDico78** [equinor/iec63131#56]
I'm thinking Valves, Tanks, Process Lines, etc. I'm wondering why so many classes have an `X` and `Y` attribute with `mm` unit and at the same time major Norsok graphical parts are ignored by the AML definition.

**Comment by cdenisey** [comment]
Because Norsok SCD AML is not meant to be used as a graphical definition tool and does not carry stencils or shapes into the coding. It is mainly used for definition of logic block and coding the relationships (or connections) between them.

**Comment by CocoDico78** [comment]
Fair enough. I'm wondering why so many classes have an `X` and `Y` attributes with `mm` unit then, if's not meant to be used as a graphical definition tool?

**Comment by ipi-equinor** [comment]
Vi sender med X og Y koordinat for å gi en hjelp til å legge ut logikken.
